Gruntfile.coffee

```diff
@@ -1,4 +1,5 @@
 path = require 'path'
+deref = require 'json-schema-deref-sync'
```

Comment:
While reading `tv4`'s [API](http://geraintluff.github.io/tv4/) I didn't find a way to dereference schemas.